### ISSUE TITLE
0.1.4 pre crashing when starting Minecraft

### ISSUE THREAD

**Issue by foenerich** [AOF-Dev/MCinaBox#417]
I'm on a OnePlus 7T with aarch64 and 8G of RAM. I set up my virtual keyboard, the runtime, user account etc. but still when I open up 1.14.4 or other versions there is a 3-6 seconds black screen and after that my app is crashing and I need to force close the app to open it again. I don't know if it has something to do with the pre release of the app or the runtime maybe someone can help me with this problem.

If you have any more questions feel free to ask them.
Log will be added.

See you
foenerich
[crash-2020-09-28_19.48.44-client.txt](https://github.com/AOF-Dev/MCinaBox/files/5294743/crash-2020-09-28_19.48.44-client.txt)

**Comment by bghyt** [comment]
1.13+ does not work, use 1.12 and below, it is a known issue.

**Comment by foenerich** [comment]
Now I can start MC but it's really lagging I can only play with like 1 FPS do you know why?

**Comment by bghyt** [comment]
Make chunks 10 or less and everything else off / low, mipmap makes everything 1FPS for me too on my OnePlus 6T so turn that off

**Comment by rApHaEL1211** [comment]
@foenerich it's crashing on mine too btw how did you fix it??

**Comment by foenerich** [comment]
> @foenerich it's crashing on mine too btw how did you fix it??

Yeah just you need to install 1.12.2 or lower everything over this version let's it crash. Also I really recommend installing forge but forge for me ist only working 1.8.9 or lower ( 1.12.2 didn't work with forge but I didn't test 1.9-1.11 so you can try). 1.8.9 forge should work fine. If you start with forge please install optifine too to get more FPS and reduce lag for me this was really helpful. :-)

**Comment by Kotarou2k4** [comment]
@foenerich forge on 1.12.2 and some other version will crash when you try to install but you can install the older forge version. I use forge 14.23.5.2847 for 1.12.2 and it worked
